deployment/terraform/dns.tf

```diff
@@ -1,26 +0,0 @@
-#
```

Comment:
As I see this being undone, I think it would be valuable to create an issue noting that when we're ready to replace what's live now with the final results of our work, we'll need to `terraform destory` against the current state.